### ISSUE TITLE
Invalid PR branch when used in repos with a leading dot

### ISSUE THREAD

**Issue by jezdez** [BetaHuhn/repo-file-sync-action#32]
**🐞 Describe the bug**

When trying to use the action from a repo whose name starts with a dot like `.github` ([community health files](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file)), it fails with git errors when it tries to create the PR branch:

```
Error: Error: Command failed: git checkout -b "repo-sync/.github/default"
fatal: 'repo-sync/.github/default' is not a valid branch name.
```

**📚 To Reproduce**

Use the action in a repo whose name wouldn't be a valid Git branch.

**💡 Expected behavior**

Invalid characters in names should be filtered out.

**🖼️ Screenshots**

**⚙️ Environment**
- Action version: v.1.5.0

**📋 Additional context**

Alternatively it would be great to be able to specify the name of the branch, or the branch suffix.

**Comment by BetaHuhn** [comment]
Hey, thanks for creating the Issue!

This is definitely a problem which needs to be fixed.

> Alternatively it would be great to be able to specify the name of the branch, or the branch suffix.

That seems to be the easiest solution and might be useful for other requirements as well. I will add an option to specify a custom branch prefix instead of the `repo-sync/REPO_NAME`.

I will start working on this tomorrow.

**Comment by BetaHuhn** [comment]
I implemented this on the [develop](https://github.com/BetaHuhn/repo-file-sync-action/tree/develop) branch.

You can try it out by using the action like this:

```yml
uses: BetaHuhn/repo-file-sync-action@develop
with:
    GH_PAT: ${{ secrets.GH_PAT }}
    BRANCH_PREFIX: custom-branch
```

I also added an [example](https://github.com/BetaHuhn/repo-file-sync-action/tree/develop#different-branch-prefix) to the README which shows how you can use the `BRANCH_PREFIX` option.

@jezdez Let me know if this resolves your issue and I will merge this into master and release a new version.

**Comment by BetaHuhn** [comment]
Friendly ping @jezdez

**Comment by jezdez** [comment]
Woops, sorry forgot to submit the form! Yep that did the trick, thank you :)

**Comment by BetaHuhn** [comment]
No problem! 😄

**Comment by BetaHuhn** [comment]
:tada: This issue is resolved in version [v1.6.0](https://github.com/BetaHuhn/repo-file-sync-action/releases/tag/v1.6.0) :tada:

To use the latest version, use the Action with the `v1` tag:

```yml
uses: BetaHuhn/repo-file-sync-action@v1
```

Or the `latest` tag:

```yml
uses: BetaHuhn/repo-file-sync-action@latest
```
